File: LottoPlayer/app/src/main/java/com/im/daeseong/lottoplayer/Util/FloatingTextView.java
package com.im.daeseong.lottoplayer.Util;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.TextView;
import com.im.daeseong.lottoplayer.R;

public class FloatingTextView {

    private View floatingview, cLClose;
    private TextView tv1, tv2;
    private Button btnClose;

    public FloatingTextView(ViewGroup viewGroup){

        floatingview = LayoutInflater.from(viewGroup.getContext()).inflate(R.layout.floating_layout, viewGroup, false);

        tv1 = (TextView)floatingview.findViewById(R.id.tv1);
        tv2 = (TextView)floatingview.findViewById(R.id.tv2);

        cLClose = (View)floatingview.findViewById(R.id.cLClose);
        cLClose.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                floatingview.setVisibility(View.GONE);
            }
        });
    }

    public View getFloatingview(){
        return floatingview;
    }

    public void setText1(String sText, int color){
        tv1.setText(sText);
        tv1.setTextColor(color);
    }

    public void setText2(String sText, int color){
        tv2.setText(sText);
        tv2.setTextColor(color);
    }
}

File: LottoPlayer(kotlin)/app/src/main/java/com/daeseong/lottoplayer/Util/FloatingTextView.kt
package com.daeseong.lottoplayer.Util

import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.constraintlayout.widget.ConstraintLayout
import com.daeseong.lottoplayer.R

class FloatingTextView(viewGroup: ViewGroup) {

    private val floatingView: View = LayoutInflater.from(viewGroup.context)
        .inflate(R.layout.floating_layout, viewGroup, false)

    private val cLClose: ConstraintLayout = floatingView.findViewById(R.id.cLClose)
    private val tv1: TextView = floatingView.findViewById(R.id.tv1)
    private val tv2: TextView = floatingView.findViewById(R.id.tv2)

    init {
        cLClose.setOnClickListener {
            floatingView.visibility = View.GONE
        }
    }

    fun getFloatingView(): View = floatingView

    fun setText1(text: String, color: Int) {
        tv1.apply {
            this.text = text
            setTextColor(color)
        }
    }

    fun setText2(text: String, color: Int) {
        tv2.apply {
            this.text = text
            setTextColor(color)
        }
    }
}